### ISSUE TITLE
Accurate of AdvancedEAST in ICDAR2015 is not good

### ISSUE THREAD

**Issue by hcnhatnam** [huoyijie/AdvancedEAST#64]
I get pretrained model in [https://github.com/gfickel/advanced_east_weights](https://github.com/gfickel/advanced_east_weights) and resul is not good 
![Screenshot from 2019-03-23 13-54-49](https://user-images.githubusercontent.com/37246426/54862883-a2d5a900-4d73-11e9-99ac-4cda76adb197.png)

**Comment by Guoth** [comment]
@hcnhatnam 
hi, can you provide the website for this test?
I want to test my model ,thank you.

**Comment by hcnhatnam** [comment]
@Guoth [Focused Scene Text 2013-2015(icdar2015)](http://rrc.cvc.uab.es/?ch=2&com=mymethods&task=1)
![Screenshot from 2019-03-23 21-42-31](https://user-images.githubusercontent.com/37246426/54867606-97ef3880-4db4-11e9-92cc-17bb5f97914f.png)
you canan dowload Offline evaluation(this is a evaluation tool).

**Comment by Guoth** [comment]
Thank you very much for your help.
Do you have any test sets about ICDAR2013 and ICDAR2011.
I try to register on the official website : http://www.cvc.uab.es/icdar2011competition/?com=contestant , but I didn't receive an email about the password, so I couldn't download the test sets.
if you have these test sets about ICDAR2013 and ICDAR2011,could you sent it to me.I'll be very grateful for your help
------------------ Original ------------------

**Comment by hcnhatnam** [comment]
you need to 1 account to access into dataset and submit. you can register in [here](http://rrc.cvc.uab.es/?com=contestant).

**Comment by Guoth** [comment]
@hcnhatnam thank you very much for your help.

**Comment by hcnhatnam** [comment]
> @hcnhatnam thank you very much for your help.

@Guoth Did yout test your model?  What is the result?

**Comment by Guoth** [comment]
@hcnhatnam
I used this repo to test the ICDAR2015 test set, but only 359 images output coordinate information, so I would like to ask how you tested the accuracy results. I don't get the result by myself.

**Comment by hcnhatnam** [comment]
Can you share model for me,then i get the result for you?email:nhatnam10a3@gmail.com

**Comment by THUHJ** [comment]
@hcnhatnam 
hi, I also tried the pre-trained model in ( https://github.com/gfickel/advanced_east_weights) on icdar 2015. My Hmeans is only around 30%.

Did you change the threshold in the cfg.py?

**Comment by hcnhatnam** [comment]
30%.. Really, no, i don't change but i used to model  in this  [model](https://github.com/gfickel/advanced_east_weights) to test

**Comment by THUHJ** [comment]
....I use the same model.....;Sad
And I also tried retrain the model based in icdar 2015 data but the result is also around 30%~40%....
Maybe I should clean anything and try again....

**Comment by THUHJ** [comment]
@hcnhatnam Has you tried fine-tune the pre-trained model based on icdar 2015 data?

**Comment by hcnhatnam** [comment]
No @THUHJ
